src/contract_cleaning.py

```diff
@@ -0,0 +1,19 @@
+def get_one_hot_encoded_contract(contract):
+    contract_as_list = contract.values.tolist()  # Returns a list of list [[]]
+    contract_cleaned = []
+
+    for item in contract_as_list:
+        contract_cleaned.append(item[0])
```

Comment:
ditch the 3 lines before this and do something like `contract_cleaned = [item[0] for item in contract_as_list]` or, if you don't need `contract_as_list` anywhere else:  `contract_cleaned = [item[0] for item in contract.values.tolist()]`

that said, i'd check if it's possible to pick up an entire slice in one go - quick stackoverflow search says the above is prettiest

again, opinion, but think it's easier on the eyeballs